app/src/main/java/com/github/sdpteam15/polyevents/database/DatabaseInterface.kt

```diff
@@ -18,7 +17,8 @@ interface DatabaseInterface {
     /**
      * Current user of this database
      */
-    val currentUser: DatabaseUserInterface?
+    val currentUser: UserEntity?
```

Comment:
I'm not sure if we should put the currentUser in the database interface and not in the user class ;)

Reply:
I would much prefer to keep global variables like currentUser and currentProfile in the same place, preferably the db, or another object. What do you think?

Reply:
It's a point of view. I still think that currentUser has nothing to do with the database. But you prefer like this, we can keep this version :)